README.md

```diff
@@ -1,2 +1,11 @@
 # Klicker Knight
+
 The spearhead thrust into the untapped market of npm cli games
+
+## Local Development
+
+```bash
+npm install
```

Comment:
Should this be `npm ci`?

Reply:
Nah. I think it's fine for the dependencies to update as we develop the project. If you run into issues you can always npm ci. Plus the CI script will catch dependency issues.